app/controllers/organizations/staff_controller.rb

```diff
@@ -1,9 +1,32 @@
 class Organizations::StaffController < Organizations::BaseController
-  layout "dashboard"
-
   before_action :require_organization_admin
+  layout "dashboard"
 
   def index
     @staff_accounts = StaffAccount.all
   end
+
+  def new
+    @user = User.new
+    @staff = StaffAccount.new(user: @user)
+  end
+
+  def create
+    @user = User.new(user_params.merge(password: SecureRandom.hex(8)))
+
+    if @user.save
+      # @user.staff_account.add_role(staff_role[:role])
```

Comment:
```suggestion
      @user.staff_account.add_role(user_params[:staff_account_attributes][:role])
```
I think this is good enough for now